TheEthicalHackerCup/Assets/AttackMinigames/Ransomware/Scripts/RansomUi.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+
+public class RansomUi : MonoBehaviour
+{
+    private RansomManager manager;
+    private TextMeshProUGUI label;
+    // Start is called before the first frame update
+    void Start()
```

Comment:
Inserting a debug.log statement here shows that this script is not being used. This might explain the `New Text` being shown in the scene